tests/model_builder/test_model_builder.py

```diff
@@ -182,7 +180,7 @@ def test_save_load(fitted_model_instance):
     test_builder2 = ModelBuilderTest.load(temp.name)
     assert fitted_model_instance.idata.groups() == test_builder2.idata.groups()
     assert fitted_model_instance.id == test_builder2.id
-    x_pred = np.random.uniform(low=0, high=1, size=100)
+    x_pred = rng.uniform(low=0, high=1, size=100)
```

Comment:
In general seeds should be local to the tests, so that they don't depend on the order or subset of tests that are run. The old tests were not seeded in the first place, which was even worse

Reply:
Ups! You are right! 🙈 let me fix this 🙏

Reply:
fixed in https://github.com/pymc-labs/pymc-marketing/pull/637/commits/f9dc0dc2371a3375ddc094d36d26ede25a665cf0